ui/components/app/confirm/info/container/container.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Box } from '../../../../component-library';
+import {
+  BackgroundColor,
+  BorderRadius,
+  Display,
+  FlexDirection,
+} from '../../../../../helpers/constants/design-system';
+
+import type { StyleUtilityProps } from '../../../../component-library/box';
+
+interface ConfirmInfoProps extends StyleUtilityProps {
+  children: React.ReactNode | React.ReactNode[];
+}
+
+export const ConfirmInfoContainer: React.FC<ConfirmInfoProps> = ({
+  children,
+  ...props
+}) => {
+  return (
+    <Box
+      borderRadius={BorderRadius.LG}
+      backgroundColor={BackgroundColor.backgroundDefault}
+      display={Display.Flex}
+      flexDirection={FlexDirection.Column}
+      {...props}
+    >
+      {children}
+    </Box>
+  );
+};
```

Comment:
The component just wraps children in a `Box`, thus I am not sure if we really need a separate component here.

Reply:
I'm not sure either. removed here [eac6663](https://github.com/MetaMask/metamask-extension/pull/22193/commits/eac666380bbb2bfdda0f7552a56700c3ab63a044)